lib/assets/models/index.js

```diff
@@ -5,7 +5,7 @@ var path      = require('path');
 var Sequelize = require('sequelize');
 var basename  = path.basename(module.filename);
 var env       = process.env.NODE_ENV || 'development';
-var config    = require(<%= configFile %>)[env];
```

Comment:
And the fact that the build is still green makes me think, that we should add a test for that